### ISSUE TITLE
QualificationRunner 9 does not work anymore

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/QualificationRunner#60]
With every project, error message `Project configuration for '<Project>' `is invalid. comes.
To reproduce:
clone e.g. the repo https://github.com/Open-Systems-Pharmacology/Raltegravir-Model and start QualiRunner via:
`QualificationRunner.exe -i "C:\Raltegravir\Evaluation_plan.json" -o "C:\Report" -p "C:\PKSim9Portable" --logLevel Debug`

This results in:
```
...
info: Qualification Runner[0]
      Starting validation runs...
dbug: Qualification Runner[0]
      Starting qualification run for project 'Raltegravir'
dbug: Qualification Runner[0]
      Qualification configuration for project 'Raltegravir' exported to 'C:\Report\temp\Raltegravir\config.json'
fail: Qualification Runner[0]
      Project configuration for 'Raltegravir' is invalid. Please check log file at 'C:\Report\temp\Raltegravir\log.txt' for details.
fail: Qualification Runner[0]
      Qualification Workflow failed
```

Log file 'C:\Report\temp\Raltegravir\log.txt' mention in the error message is not created.
The whole content of the output folder created till the crash is here: [out.zip](https://github.com/Open-Systems-Pharmacology/QualificationRunner/files/4521587/out.zip)

This happens with the latest versions of QualiRunner 9 (9.0.57) and PK-Sim 9 portable (9.0.137)

**Comment by Yuri05** [comment]
Ok, the error above does not appear with PK-Sim portable **9.0.137** (probably because of the fix https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1286 ). 
However another error comes now (during or after observed data sets are exported):
```
dbug: Raltegravir[0]
      Observed data 'Iwamoto 2008 - 1600mg - Raltegravir - PO - 1600 mg - Plasma - agg. (n=24)' exported to 'C:\CompletSetup_SBSuite\OSPSuite_Setups\OSP_Suite_9.0\tmp\out\ObservedData\Iwamoto 2008 - 1600mg - Raltegravir - PO - 1600 mg - Plasma - agg. (n=24).xlsx'
fail: Qualification Runner[0]
      Project configuration for 'Raltegravir' is invalid. Please check log file at 'C:\CompletSetup_SBSuite\OSPSuite_Setups\OSP_Suite_9.0\tmp\out\temp\Raltegravir\log.txt' for details.
fail: Qualification Runner[0]
      Qualification Workflow failed
```

Log file does not contain any hint what goes wrong: [log.txt](https://github.com/Open-Systems-Pharmacology/QualificationRunner/files/4521996/log.txt)

And: if instead of the portable version I use the latest **development** version of PK-Sim: it works!
So either something is missing in the portable setup.. or I have no idea.

**Comment by msevestre** [comment]
Looks like smart xls Problem to me.  Are we exporting XLS file??
Does it work with other obs data?

**Comment by Yuri05** [comment]
>Looks like smart xls Problem to me. Are we exporting XLS file??

Yep, could be. We do export both xls and csv (and I think xls is used just for user convenience (?), because RE imports CSV only). And if I look now at the created ObservedData folder, I see only CSVs. So yes, SmartXLS could be a problem.

**Comment by msevestre** [comment]
Can you copy PKSim.exe.confug and rename it to PKSim.CLI.exe.config in the portable version. Does it work then?

**Comment by Yuri05** [comment]
>Can you copy PKSim.exe.confug and rename it to PKSim.CLI.exe.config in the portable version. Does it work then?

Yep. that worked!
